### ISSUE TITLE
Twilio not Calling our server's Voice Endpoint

### ISSUE THREAD

**Issue by nkarman** [twilio/voice-quickstart-objc#43]
Hello @bchen-twilio , My team and I are trying to connect VoIP calls using Twilio, but Twilio never calls our /Voice endpoint for TwiML instructions, or seems to attempt to. I have included verbose console output in a pastebin below. No events or errors are visible on the Twilio Dashboard. Thanks in advance for your help!

https://pastebin.com/ufr6D4ct

**Comment by bobiechen-twilio** [comment]
Hi @nkarman 

Sorry that you and the team are experiencing this.
From the SDK log I did see a `500 Server Internal Error` which I guess was caused by the identity embedded in the access-token. I would suggest that you don't have whitespace in the client identity. Since the Voice mobile SDK is still relying on the SIP signaling protocol, the client identity should contain only alpha-numeric and underscore characters.

I just tried using `%20` in the identity and generate an access token. The token gave me the same `500 Server Internal Error` when making outbound calls.

I'll escalate this to the backend team and ask them to at least surface the error to user dashboard instead of just ignoring the exception and returning a less-informative error code.

Let me know :)
-bobie
